sponsors/forms.py

```diff
@@ -48,16 +49,23 @@ class SponsorshiptBenefitsForm(forms.Form):
         required=False,
         empty_label=None,
     )
+    add_ons_benefits = PickSponsorshipBenefitsField(
+        required=False,
+        queryset=SponsorshipBenefit.objects.add_ons().select_related("program"),
+    )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        benefits_qs = SponsorshipBenefit.objects.select_related("program")
+        benefits_qs = SponsorshipBenefit.objects.with_packages().select_related(
+            "program"
+        )
+
         for program in SponsorshipProgram.objects.all():
             slug = slugify(program.name).replace("-", "_")
             self.fields[f"benefits_{slug}"] = PickSponsorshipBenefitsField(
                 queryset=benefits_qs.filter(program=program),
                 required=False,
-                label=_(f"{program.name} Sponsorship Benefits"),
+                label=_(f"{program.name} Benefits"),
```

Comment:
gettext extraction will get the source string, but at runtime the f-string will be evaluated before it’s passed to gettext so this won’t be translated.
```suggestion
                label=_("{program_name} Benefits").format(program_name=program.name),
```